browser/src/control/jsdialog/Widget.TreeView.js

```diff
@@ -362,7 +379,7 @@ function _headerlistboxEntry(parentContainer, treeViewData, entry, builder) {
 			td.innerText = entry.columns[i].text;
 
 		if (!disabled)
-			$(td).click(clickFunction);
+			$(parentContainer).click(clickFunction);
```

Comment:
why is that needed?

Reply:
td only covers the label part of the option. it does not fire the `clickfunction` if we do check/uncheck the radio button.

`parentContainer` will cover that case where if user click on label or on checkbox, it will fire the `clickFunction`